leonardo-example.conf

```diff
@@ -0,0 +1,119 @@
+# Example config file for leonardo.
+# Anything commented out here will have have defaults specified or is not necessary to run leo.
+# All values can be modified for your specific google environment, database, and security certificates
+
+
+# Google Cloud dataproc configuration
+dataproc {
+  #applicationName = "leonardo"
+  dataprocDefaultRegion = "YOUR_REGION"  # the google region for your dataproc
+  leoGoogleProject = "GOOGLE_PROJECT"    # the name of the google project to use during cluster startup
+                                         # *this is not the project the cluster will be created in
+  clusterUrlBase = "https://YOUR_DOMAIN/notebooks" # the base url to access your cluster
+  #jupyterServerName = "jupyter-server"
+}
+
+# These are the cluster defaults that have already been configured.
+# You should only set them in your config if you want a different value
+#clusterDefaults {
+  #numberOfWorkers = 0            # Standard, non-preemptible workers; Can't be 1
+  #masterMachineType = "n1-standard-4"
+  #masterDiskSize = 500           # Minimum is 100
+  #workerMachineType = "n1-standard-4"
+  #workerDiskSize = 500           # Minimum is 100
+  #numberOfWorkerLocalSSDs = 0    # Can be 0-8
+  #numberOfPreemptibleWorkers = 0 # Must be 0 if number of workers is 0
+#}
+
+# Database connection information
```

Comment:
Maybe a note on DB requirements and initialization would be helpful here. I realize this will probably be documented elsewhere in the future, but for now even general notes would be helpful.

Reply:
+1. We run liquibase migrations on startup so there is no schema initialization needed. I _think_ a database named `leonardo` needs to exist (liquibase won't create the DB if it doesn't exist). We use [slick](http://slick.lightbend.com/) as a database access library. Other DB parameters and default values can be seen [here](http://slick.lightbend.com/doc/3.2.1/api/index.html#slick.jdbc.JdbcBackend$DatabaseFactoryDef@forConfig(path:String,config:com.typesafe.config.Config,driver:java.sql.Driver,classLoader:ClassLoader):JdbcBackend.this.Database).